uwmac0001.xml

```diff
@@ -5794,6 +5808,21 @@
           <abstract>Additional personnel files dating from 1960-2011. </abstract>
         </did>
       </c01>
+      <c01 level="otherlevel" otherlevel="minimal" id="acc2016-032">
+        <did>
+          <unitid>Accession 2016-032</unitid>
+          <unittitle>Departmental Files, Meeting Minutes, and Personnel Files
+              <unitdate>1968-2013</unitdate></unittitle>
+          <physdesc>
+            <extent>13.6 cubic ft. (16 boxes)</extent>
+          </physdesc>
+          <abstract>The first 10 boxes consist of records from various departments (mostly financial
+            statistics and correspondence) within the college as well as meeting minutes from several committees and
+            subcommittees. Boxes 8-10 contain unprocessed pocket folders of handwritten meeting
+            minutes from several different Letters and Science committees. The last 6 boxes are
+            personnel files. </abstract>
+        </did>
+      </c01>
```

Comment:
I assume you just haven't had time to enter it yet, but just in case, as long as we have the preliminary inventory we should enter it into the EAD. There's some slight differences for entering minimal-processing contents, which we'll go over tomorrow.